src/Nethermind/Nethermind.Runner.Test/Ethereum/Steps/Migrations/ReceiptMigrationTests.cs

```diff
@@ -160,7 +160,8 @@ public void RemoveReceipts(Block block)
             }
 
 #pragma warning disable CS0067
-            public event EventHandler<BlockReplacementEventArgs> ReceiptsInserted;
+            public event EventHandler<BlockReplacementEventArgs> NewCanonicalReceipts;
+            public event EventHandler<ReceiptsEventArgs> ReceiptsInserted;
```

Comment:
```suggestion
            public event EventHandler<BlockReplacementEventArgs>? NewCanonicalReceipts;
            public event EventHandler<ReceiptsEventArgs>? ReceiptsInserted;
```

Reply:
This one complains that nullability is not enabled for the project, should I toggle it on?

Reply:
Nah, enabling it generates too many errors for this PR to resolve :sweat_smile: